packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -443,6 +447,49 @@ const FolderDataGrid = ({
             hideSortIcons: true,
             disableColumnMenu: true,
         },
+        ...(enableLicenseFeature
+            ? ([
+                  {
+                      field: "license",
+                      headerName: intl.formatMessage({
+                          id: "comet.dam.file.license",
+                          defaultMessage: "License",
+                      }),
+                      headerAlign: "left",
+                      align: "left",
+                      minWidth: 200,
+                      renderCell: ({ row }) => {
+                          if (isFile(row) && row.license && row.license.type) {
+                              return (
+                                  <GridCellContent
+                                      primaryText={licenseTypeLabels[row.license.type]}
+                                      secondaryText={
+                                          row.license.expiresWithinThirtyDays || row.license.hasExpired ? (
+                                              <LicenseValidityTags
+                                                  {...row.license}
+                                                  expirationDate={row.license.expirationDate ? new Date(row.license.expirationDate) : undefined}
+                                              />
+                                          ) : (
+                                              <div>
```

Comment:
Could this be a React Fragment instead?

Reply:
Yes 36b04bdb78916548695bba52dbb1d12eb9c5dbc6